### ISSUE TITLE
`create_sparklines()` creates invalid sparklines if `sheet = current_sheet()`

### ISSUE THREAD

**Issue by elipousson** [JanMarvin/openxlsx2#1206]
When sparklines are created using the default value of `current_sheet()`, it appears that it creates a malformed file.

The following example creates a file using `sheet = current_sheet()`:

``` r
library(openxlsx2)

sparklines <- c(
  create_sparklines(current_sheet(), "A3:L3", "M3", type = "column", first = "1"),
  create_sparklines(current_sheet(), "A2:L2", "M2", markers = "1"),
  create_sparklines(current_sheet(), "A4:L4", "M4", type = "stacked", negative = "1")
)

t1 <- AirPassengers
t2 <- do.call(cbind, split(t1, cycle(t1)))
dimnames(t2) <- dimnames(.preformat.ts(t1))

wb <- wb_workbook()$
  add_worksheet()

wb <- wb$
  add_data(x = t2)$
  add_sparklines(sparklines = sparklines)

wb
#> A Workbook object.
#>  
#> Worksheets:
#>  Sheets: Sheet 1 
#>  Write order: 1

packageVersion("openxlsx2")
#> [1] '1.11.0.9000'
```

<sup>Created on 2024-12-08 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

When I open the file, I see the following warning/error message:

<img width="275" alt="Screenshot 2024-12-08 at 4 19 34 PM" src="https://github.com/user-attachments/assets/d763c0f2-d473-4097-a3e1-b6ca885544f4">

When I open the file the first sparkline appears but no others:

<img width="211" alt="Screenshot 2024-12-08 at 4 20 24 PM" src="https://github.com/user-attachments/assets/59f7bf10-c2e7-4372-b9fa-ff601aa60af6">

**Comment by JanMarvin** [comment]
Hi @elipousson , that's probably because I took inspiration from `create_comment()`, when I added `create_sparklines()` and didn't add the functionality into `wb_add_sparklines()`. Since `create_sparklines()` is just a basic XML creating function, it does not know about the current sheet. Contrary to workbook wrappers. To make this work, we probably would have to create some arbitrary sheet name that could be replaced when adding the sparkline into the worksheet. I'll try to have a look.
The simple fix would be to add a notification to the documentation that using the waiver outside of workbook functions is just not supported.
